File: 2 SEMESTER/Object Oriented Programming/Projects/PROJECT2_JAVA/src/Organisms/Animals/Antelope.java
package Organisms.Animals;

import Game.World;
import com.sun.source.tree.ReturnTree;

import java.awt.*;
import java.io.Serializable;

public class Antelope extends Animal implements Serializable {
    public Antelope(World world, int x, int y) {
        super("A", "Antelope", new Color(87, 49, 11), 4, 4, 2, world,x,y);
    }
    @Override
    public Antelope getChild(int x, int y)
    {
        return new Antelope(world,x,y);
    }
    @Override
    public boolean IsEscaping()
    {
        return world.getRandomNumber(2) == 0;
    }
}

File: 2 SEMESTER/Object Oriented Programming/Projects/PROJECT3_PYTHON/Organisms/Animals/antelope.py
from Organisms.Animals.animal import Animal


class Antelope(Animal):
    def __init__(self, position_x, position_y, world):
        super().__init__(position_x, position_y, "Antelope", 'A', 4, 4, world, 2, (139, 69, 19))
